ext/MitsubaLoader/CMitsubaLoader.h

```diff
@@ -224,7 +265,9 @@ class CMitsubaLoader : public asset::IAssetLoader
 		CMitsubaLoader(asset::IAssetManager* _manager);
 
 	protected:
-		asset::IAssetManager* manager;
+		asset::IAssetManager* m_manager;
+		//TODO need one packer per format class
+		core::smart_refctd_ptr<asset::ICPUTexturePacker> m_texPacker;
```

Comment:
put in some metadata, either global or pipeline, you'll want to return that with the asset

Reply:
VT textures (both) are getting returned as ds0 through metadata

Reply:
these are just the textures, and not the texture manager.

Also for multithreading the texture packer cannot be global and persistent between loading calls

Reply:
oh, i thought i put it into SContext. Sure, thanks

Reply:
> these are just the textures, and not the texture manager.

yea, but packer is useful as long as something is being packed

Reply:
would be good to return it though in the metadata, it doesn't cost anything really.

Reply:
ok